src/libraries/Common/tests/System/GenericMathHelpers.cs

```diff
@@ -392,31 +400,35 @@ public static class ParsableHelper<TSelf>
     }
 
     public static class PowerFunctionsHelper<TSelf>
-        where TSelf : IPowerFunctions<TSelf>, INumberBase<TSelf>
+        where TSelf : IPowerFunctions<TSelf>
     {
         public static TSelf Pow(TSelf x, TSelf y) => TSelf.Pow(x, y);
     }
 
     public static class RootFunctionsHelper<TSelf>
-        where TSelf : IRootFunctions<TSelf>, INumberBase<TSelf>
+        where TSelf : IRootFunctions<TSelf>
     {
         public static TSelf Cbrt(TSelf x) => TSelf.Cbrt(x);
 
+        public static TSelf Hypot(TSelf x, TSelf y) => TSelf.Hypot(x, y);
+
+        public static TSelf RootN(TSelf x, int n) => TSelf.RootN(x, n);
+
         public static TSelf Sqrt(TSelf x) => TSelf.Sqrt(x);
     }
 
-    public static class ShiftOperatorsHelper<TSelf, TResult>
-        where TSelf : IShiftOperators<TSelf, int, TResult>
+    public static class ShiftOperatorsHelper<TSelf, TOther, TResult>
+        where TSelf : IShiftOperators<TSelf, TOther, TResult>
```

Comment:
Re: my other comment, can you elaborate on this change? I can't tell if this is related to the API review.

Reply:
Same as https://github.com/dotnet/runtime/pull/71800#discussion_r917221834, this is a test only helper and it was missed as part of last weeks change. This just ensures its up to date with everything else.